src/main/java/htsjdk/samtools/SAMSequenceRecord.java

```diff
@@ -139,6 +152,49 @@ private void setSequenceName(final String name) {
     // Private state used only by SAM implementation.
     public void setSequenceIndex(final int value) { mSequenceIndex = value; }
 
+    /** Returns a set with alternative sequence names. Modifications will not affect the record instance. */
+    @XmlAttribute(name = "alternative_sequence_names")
+    public Set<String> getAlternativeSequenceNames() {
+        final String anTag = getAttribute(ALTERNATIVE_SEQUENCE_NAME_TAG);
+        return (anTag == null) ? new LinkedHashSet<>() : new LinkedHashSet<>(Arrays.asList(anTag.split(ALTERNATIVE_SEQUENCE_NAME_SEPARATOR)));
```

Comment:
This shouldn't return a modifiable set, should it? E.g. I would use `Collections.emptySet()` for the `null` case.